### DISCUSSION TITLE
`cargo install` without `--force`?

### DISCUSSION THREAD

**Discussion by mightyiam** [sagiegurari/cargo-make#654] [Q&A] [answered]
The `--force` flag ignores a possibly installed crate. That takes more time. I tried setting `min_version` but that didn't seem to disable `--force`. Am I missing something?

Even `install_command` cannot prevent `--force`, it seems. Is there any way to prevent `--force`? I fell like perhaps `--force` should be used only when `min_version` or `version` are specified and the currently installed version does not satisfy.

**Comment by sagiegurari** [comment, marked as answer]
cargo-make doesn't run install if crate already exists.
so to better understand best to provide a test case.

**Reply by mightyiam** [reply]
Thank you. I suppose I did something funny. In any case, I'm using https://crates.io/crates/cargo-run-bin now, which seems to solve the problem of dependencies earlier.
